examples/plot_simulate_somato.py

```diff
@@ -3,20 +3,21 @@
 05. Source reconstruction and HNN
```

Comment:
This might be a little wordy, but what do you think of `05. From sensor to simulation: MEG source reconstruction with HNN` so that the title is more descriptive?

Reply:
+1 for more descriptive title. However, "MEG source reconstruction with HNN" is a bit inaccurate ... because HNN does not do source reconstruction. Maybe:

05. From MEG sensor-space data to HNN simulation

? Any other title that's not inaccurate will do :)

Reply:
I'm not sure I understand what is inaccurate about it. "Source reconstruction" is what we had originally.

Reply:
I think @jasmainak means it implies that `hnn_core` is doing the reconstructing. I'm not too keen on the 'From' though, since these are really two completely separate code sequences tacked after one another. How about?

```suggestion
05. Comparing MEG source reconstruction to HNN model prediction
```

Reply:
Sorry, I just noticed the most recent comment now after reloading page. I actually don't think of the MNE inverse solution and HNN simulation as "completely separate code sequences" at all. In fact, very much the opposite. What we're trying to demonstrate is a process that most users won't find intuitive. Fitting sensor-level data into the HNN framework is a process that requires interdependent considerations between the inverse solution technique and HNN, and the only reason it appears simple in this example is because we've left out the narrative about how I got the specific results and values that we present in this example. I'm not committed to any title in particular, although @jasmainak's suggestion is growing on me. Big picture: we should find a way to communicate that this is a process rather than an end result to the user.